src/main/java/lotto/step2/domain/WinningNumbers.java

```diff
@@ -0,0 +1,54 @@
+package lotto.step2.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class WinningNumbers {
+
+    private final static int MATCHED = 1;
+    private final static int MISMATCHED = 0;
+    private final static int LOTTO_SIZE = 6;
+    private final static String ILLEGAL_LOTTO_NUMBERS = "로또는 6개의 서로 다른 숫자로 만들어져야 합니다";
+    private final static String ILLEGAL_BONUS_NUMBER = "보너스 번호는 6개 로또 번호와 다른 숫자여야 합니다.";
+    private final Set<LottoNumber> lottoNumbers;
+    private final LottoNumber bonusNumber;
+
+    public WinningNumbers(String[] numbers, String bonusNumber) {
+        this(new ArrayList<LottoNumber>(Arrays.stream(numbers)
```

Comment:
```suggestion
        this(new ArrayList<>(Arrays.stream(numbers)
```

굳이 수정을 요구하는것은 아니지만, 이미 LottoNumber 로 암시적으로 변형 가능할 때에는 generic 부분의 타입을 명시하지 않아도 괜찮습니다.
IntelliJ IDE 에서 warning 을 보여줘서 언급을 해봅니다.